nmrfx-processor/src/main/resources/pyproc.py

```diff
@@ -1749,6 +1773,32 @@ def DX(disabled=False, vector=None, process=None):
     op = Dx()
     return op
 
+def SUPPRESS(winSize=31, shift='0.0f',disabled=False, vector=None, process=None):
+    ''' Time domain signal suppression.
+    Parameters
```

Comment:
SCALING: The updating of the property sheet (showing parameters) when deleting any list entry is now fixed.
I added (in version tested above) an autoscale operation so that every time you change an operation the display is autoscaled.  But I have now decided I don't like that because the effect of some operations (TDCOMB, APODIZE etc.) that change the intensity of the spectrum is no longer clear.  So the new version only autoScales if the frequency domain status is changed (adding or removing an FT).  That's what had originally bothered me before (adding FT and the scale would go way up or down).  So I think this new mode is a good compromise.

SUPPRESS: I wanted the operations in the common section to have somewhat meaningful names. But, we can't change names of existing ops or we'll break existing scripts, so I added a new op.  I think (or hope) that SUPPRESS is more meaningful as a name than TDSS.  SIGNAL_SUPPRESSION would be more clear, but I didn't want the names to be too long.  But, suggestions welcomed.  The description changed because really it can be used for suppressing any signal, not just solvent.  I tried to simplify the common operation parameters so removed nPasses as I have never actually had a reason to change that parameter.